systems/crusher_quick_start_guide.rst

```diff
@@ -1340,33 +1340,52 @@ For example:
 Theoretical Roofline
 ^^^^^^^^^^^^^^^^^^^^
 
-The theoretical (attainable) roofline constructs a theoretical maximum performance for each operational intensity.
+The theoretical (this is NOT attainable FLOPS/s) roofline constructs a theoretical maximum performance for each operational intensity.
 The theoretical roofline can be constructed as:
 
 .. math::
 
     FLOPS_{peak} = minimum(OpIntensity * BW_{HBM}, theoretical\_flops)
 
-On Crusher, the memory bandwidth for HBM is 1.6 TB/s, and the theoretical peak flops is calculated by:
+
+On Crusher, the memory bandwidth for HBM is 1.6 TB/s, and the theoretical peak floating-point FLOPS/s is calculated by:
 
 .. math::
 
     TheoreticalFLOPS = 128 FLOP/cycle/CU * 110 CU * 1700000000 cycles/second = 23.9 TFLOP/s
 
 
+However, when using MFMA instructions, the theoretical peak floating-point FLOPS/s is calculated by:
+
+.. math::
+
+    TheoreticalFLOPS = 256 FLOP/cycle/CU * 110 CU * 1700000000 cycles/second = 47.8 TFLOP/s
+
+
+.. note::
+    Attainable rooflines are constructed using microbenchmarks, and are not currently discussed here.
```

Comment:
In this "note" section, I think some context should be given about "attainable" rooflines. While it's defined here, it's not clear when/where it might be used.